dependencies.rosinstall

```diff
@@ -1,4 +1,4 @@
 - git:
     uri: https://github.com/Gepetto/example-robot-data.git
-    version: master
+    version: devel
```

Comment:
Is there a reason why we need to use this devel branch?

Reply:
Because we need a fix from the devel branch until there is another example-robot-data release